cax/cax.json

```diff
@@ -3,76 +3,78 @@
     "name": "xenon1t-daq",
```

Comment:
yes, but we have to use the cax-mv on all the data.  I'm still waiting for a computing meeting so we can discuss directory layout.  After this merge, we'll be ready for switching to data manager (I think SSH is open to all sites?).  It's just every time I test it, there's a small thing we have to fix.